monitor.go

```diff
@@ -163,24 +171,30 @@ func (p *stageMonitor) draw(dc drawContext) {
 	default:
 		x, y := int(p.x), int(p.y)
 		labelRender := gdi.NewTextRender(defaultFont2, 0x80000, 0)
+		labelRender.Scale = p.size
 		labelRender.AddText(p.label)
 		labelW, h := labelRender.Size()
 
 		textRender := gdi.NewTextRender(defaultFontSm, 0x80000, 0)
+		textRender.Scale = p.size
 		textRender.AddText(val)
 		textW, textH := textRender.Size()
 		textRectW := textW
-		if textRectW < stmDefaultSmW {
-			textRectW = stmDefaultSmW
+		if textRectW < int(stmDefaultSmW*p.size) {
+			textRectW = int(stmDefaultSmW * p.size)
 		}
-		w := labelW + textRectW + (stmHoriGapSm * 3)
-		h += (stmVertGapSm * 2)
+		hGap := stmHoriGapSm * p.size
```

Comment:
There's issue with style details here. For example:

* The gap size and rect height is incorrect
* The label color is not correct
* There should be no out border for the round-rect
* There should be background of `#fff`
* The background of value-part is translucent, which is not expected
* There should be no border of round-rect of value-part
* The width of value-part is not right

![image](https://github.com/user-attachments/assets/285f9771-837c-4669-a4a4-6b919e4fc41b)

<img width="714" alt="image" src="https://github.com/user-attachments/assets/fccfa3ae-adda-40ae-a1f3-b0b5dc53d3c1">

Reply:
The issues related to font size, font changing, and layout adaptation will be resolved after the font library implementation is upgraded.